### ISSUE TITLE
Failed to resolve entry for package "react-model-view-viewmodel".

### ISSUE THREAD

**Issue by Crim** [Andrei15193/react-model-view-viewmodel#14]
Hello!  
I'm getting the following error when trying to replicate your examples here: https://codesandbox.io/examples/package/react-model-view-viewmodel

```
5:49:33 PM [vite] (client) Pre-transform error: Failed to resolve entry for package "react-model-view-viewmodel". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: vite:import-analysis

"use strict";
  3  |  import { ViewModel } from "react-model-view-viewmodel";
     |                             ^
  4  |  export class StaticTextConfigViewModel extends ViewModel {
  5  |    _textValue = "";
```

Any ideas?

From Package.json:

```
"dependencies": {
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "react-model-view-viewmodel": "^3.0.0"
  },
```

Thanks!

**Comment by Andrei15193** [comment]
Hi, thanks for reaching out!

I'll have to check, so far I haven't encountered any issues but I'm most likely missing something 🤔

What setup/tooling are you using?

**Comment by Crim** [comment]
I'm using a basic out of the box setup using vite.

Thanks!

**Comment by Andrei15193** [comment]
I managed to reproduce the issue even with basic webpack setup. On other setups I also have TypeScript configured which may explain why it works on CodeSandbox.

On further investigation it seems to be related to the `module` parameter in `package.json`, I am unsure why I added this in the first place to be honest.

I'll be publishing a fix shortly, just need to run a few more tests.

**Comment by Crim** [comment]
awesome thanks! No rush, but appreciate the help.

Excited to dive into this, your docs look great for getting people started with it.

**Comment by Andrei15193** [comment]
Yep, with version 3 I've added a lot more documentation and managed to generate it using [typedoc](https://typedoc.org/). Everything that is in the API docs is picked from source code and JSdoc comments. There are other goodies with this version that makes it an overall improvement compared to the previous one.

I've published version `3.0.1` with the fix, from what I tested it works with JS-only configs as well as TS configs. I've also updated the samples on CodeSandbox to use the latest version.

If anything comes up, please let me know! Happy coding!

**Comment by Crim** [comment]
rad, seems to work, appreciate it!
